pkgs/development/libraries/lief/default.nix

```
@@ -1,8 +1,62 @@
-{ lib, fetchzip }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, python
+, cmake
+}:
 
-fetchzip {
-  url = "https://github.com/lief-project/LIEF/releases/download/0.9.0/LIEF-0.9.0-Linux.tar.gz";
-  sha256 = "1c47hwd00bp4mqd4p5b6xjfl89c3wwk9ccyc3a2gk658250g2la6";
+let
+  pyEnv = python.withPackages (ps: [ ps.setuptools ]);
+in
+stdenv.mkDerivation rec {
+  pname = "lief";
+  version = "0.11.4";
+
+  src = fetchFromGitHub {
+    owner = "lief-project";
+    repo = "LIEF";
+    rev = version;
+    sha256 = "DgsTrJ2+zdXJK6CdDOan7roakaaxQiwrVeiQnzJnk0A=";
+  };
+
+  outputs = [ "out" "py" ];
```

```
lief: Ask if output path include (2.6M) could be split with outputs = [ ... "dev" ];
lief: Ask if output path lib (46.9M) could be split with outputs = [ ... "lib" ];
```

```suggestion
  outputs = [ "out" "py" "dev" ];
```

Maybe we could also add lib or delete the static file.

```
$ tree -l results/lief/lib/
dr-xr-xr-x root root 47 MB Thu Jan  1 01:00:01 1970  lib
├── .r--r--r-- root root  47 MB Thu Jan  1 01:00:01 1970  libLIEF.a
└── dr-xr-xr-x root root 4.3 KB Thu Jan  1 01:00:01 1970  pkgconfig
   └── .r--r--r-- root root 347 B Thu Jan  1 01:00:01 1970  LIEF.pc
```

---

I don't want to split these up because we disable the cmakeConfigure stuff, so the generated pkgconfig won't pick up the prefix (I had to patch this in manually).

I also don't want to delete the static file because I don't know if it's used by anything.